### ISSUE TITLE
Calling (Video/Audio) automatically get disconnected within 1 minute

### ISSUE THREAD

**Issue by anksy** [MidEndProject/node-apprtc#5]
I've installed the setup on my local environment. It working like a charm. But I'm having a serious issue when i am on call with any user, it automatically disconnected within 1 minute everytime. Can you please check it once or is there any configuration I've to do?

**Comment by fitraditya** [comment]
Hi,

Did you use the demo link or use your own deployed app?

**Comment by anksy** [comment]
Yes, I have checked the demo link, but I was getting this issue in apprtc ios/android sdk. So I was wondering whether this is server side issue or what? Also can you confirm me the steps to have my own TURN server. I have created one, but not so sure whether its implemented in correct way or not. Thanks

**Comment by fitraditya** [comment]
I suggest that you use your own STUN/TURN server. You can install Coturn or if you use Ubuntu, you can install resiprocate-turn-server. To verify and test STUN/TURN service, checkout this link https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/.

And the last part is modify node-apprtc. This application use TURN API to get TURN url and credential. If you want to use your own STUN/TURN server, you can modify this part https://github.com/MidEndProject/node-apprtc/blob/master/controller/index.js#L20.

Hope it will help.

**Comment by NeelamParmar** [comment]
Hello,

We have created our own STUN and TURN servers.
We created using windows, but faced issue for IceServer connection while connecting from mobile app.
So we installed **TURN server on Ubuntu and magically it worked** :)

But still we faced 1 minute issue while connecting from mobile app. The server disconnects within a minute or less.

** @fitraditya : Do I need to change anything in server configurations ?** 

> We also verfied using https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/.

**Comment by NeelamParmar** [comment]
Can wss_host_port_pair can be the issue for disconnection within 1 minute ?

In code, this one is used : "node-apprtc-ws.herokuapp.com'

Do we need our own ?
